backend/app/server/database.py

```diff
@@ -1,6 +1,6 @@
 import motor.motor_asyncio
 
-MONGO_DETAILS = "mongodb://localhost:27017"
+MONGO_DETAILS = "mongodb+srv://sociomark:sociomark@userdb.7mwfe.mongodb.net/users?retryWrites=true&w=majority"
```

Comment:
In any case, we should use the `.env` file to store this URL (even if it's localhost)